templates/wip-card.html

```diff
@@ -0,0 +1,25 @@
+<div class="column">
+    <div class="card">
+        <div class="card-content">
+            <div class="media">
+                <div class="media-left">
+                    <p style="font-size: 2rem;"><i class="fa {{ card.icon_slug }}"></i></p>
+                </div>
```

Comment:
Would be nice if we could set a standard size for these instead of having them scale based on contents. Right now all three on the preview are different.